src/components/webview/sourceControl.ts

```diff
@@ -9,6 +9,11 @@ interface UserAccountStorage {
 	getUserAccount(): string | null;
 }
 
+type CreateIssueResponse = {
+	url: string;
```

Comment:
what's the difference between `url` and `html_url`?

Reply:
`url` is the path of GET api endpoint ("https://api.github.com/repos/DmytroHryshyn/test_repo/issues/31"), and `html_ url` is normal url. I think we dont need `url` at all, i will remove it